apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -14,6 +14,38 @@ import { colors } from '@condo/domains/common/constants/style'
 
 type IAddressSearchInput = SelectProps<string>
 
+export const renderOption = (dataItem, searchValue) =>  (
```

Comment:
What is this change for? Is there performance or optimization issues? need to be reverted

Reply:
i need same function in custom PropertyAddressSearchInput, if i wrap this function in useCallback how it was before, is it ok?

Reply:
Add local implementation of renderOption for your custom search input.

Reply:
Even if I need the exact same renderOption?